src/common/isolatable_hardwares.cpp

```diff
@@ -318,6 +311,12 @@ std::optional<devtree::DevTreePhysPath> IsolatableHWs::getPhysicalPath(
             return std::nullopt;
```

Comment:
I am planning to push separate PR to create the PEL during guard operations. This PR scope is "use interface name instead of an object name" to support TPM targets.

FYI, we have already covered PEL for reconfiguration use case similarly, planning to push for guard operation as well.